crates/forge/src/cmd/build.rs

```diff
@@ -56,10 +56,10 @@ pub struct BuildArgs {
     #[serde(skip)]
     pub sizes: bool,
 
-    /// Ignore initcode contract bytecode size limit introduced by EIP-3860.
-    #[arg(long, alias = "ignore-initcode-size")]
+    /// Ignore initcode contract bytecode size limit.
+    #[arg(long, alias = "ignore-eip-3860")]
     #[serde(skip)]
-    pub ignore_eip_3860: bool,
+    pub ignore_initcode_size: bool,
```

Comment:
@zerosnacks @DaniPopes are you okay with changing the default here? Shouldn't be any breaking changes from cli usage due to this